evaluation/evaluate_text_generation.py

```diff
@@ -1,14 +1,18 @@
 import numpy as np
 from datasets import load_dataset
-from sacrebleu import corpus_bleu
 from transformers import pipeline
-
+from rouge_score import rouge_scorer
 from dataset import KeyValueDataset
 from tasks.TaskTypes import TaskType
 
+scorer = rouge_scorer.RougeScorer(['rouge1'])
+
 
-def sacrebleu_score(hypotheses, references):
-    return corpus_bleu(hypotheses, [references]).score
+def rogue_score(hypotheses, references):
+    total_f1_scores = 0.0
+    for ref, hyp in zip(references, hypotheses):
+        total_f1_scores+=scorer.score(ref, hyp)['rouge1'][2] # [2] for f1 score
+    return total_f1_scores/len(hypotheses)
```

Comment:
The previous code supported multi-reference while this one does not.

Reply:
yes, earlier library function "score" from "sacrebleu" is taking a list of hypotheses and references as input, "score" function in "rouge" is taking one hypothesis, reference pair as input. Please let me know if somehow we can make it multi-reference?

Reply:
Yes, the original ROUGE paper describes how to do this. I implemented this in BIG-bench like this: https://github.com/google/BIG-bench/blob/main/bigbench/api/task_metrics.py#L86